exclusion.dic

```diff
@@ -0,0 +1,68 @@
+valerie
+ansi
+specialname
+awaitable
+csproj
+Usings
+warnaserror
+lpfl
+cpuid
+retq
+vmax
+decryptor
+Abidal
+Autoproperty
+Barthez
+Coupet
+Landreau
+Boumsong
+Chimbonda
+Gallas
+Givet
+Sagnol
+Thuram
+Dhorasoo
+Diarra
+Makelele
+Djibril Cisse
+Saha
+Trezeguet
+Wiltord
+Saha
+Zaccardo
+Zambrotta
+Cannavaro
+Oddo
+Gattuso
+Camoranesi
+Gilardino
+Iaquinta
+beforefieldinit
+mscorlib
+Dever
+Stokesbary
+Heavey
+enumtest
+Equatable
+Fireswamp
+spokane
+urls
+iassist
+worldvision
```

Comment:
Is it possible to drop this and fix usage to be `worldVision`?

Reply:
I made the change in the most recent commit, but did it separately if we want to roll it back